.github/scripts/hermetic_library_generation.sh

```diff
@@ -104,6 +107,7 @@ docker run \
   -u "$(id -u):$(id -g)" \
   -v "$(pwd):${workspace_name}" \
   -v "$HOME"/.m2:/home/.m2 \
+  -e generator_version="${generator_version}" \
```

Comment:
From our [shell style guide](https://engdoc.corp.google.com/eng/doc/devguide/shell/styleguide.md?cl=head#s7.3-constants-and-environment-variable-names): _Constants and anything exported to the environment should be capitalized, separated with underscores, and declared at the top of the file._

The _top of the file_ part doesn't apply, but I think it's reasonable to make it uppercase for consistency with other env vars we set up in the Dockerfile

Reply:
Makes sense, do you mind updating it in place of Joe?

Reply:
Done